src/nci_adapter.c

```diff
@@ -430,7 +579,7 @@ nci_adapter_nci_intf_activated(
             tag = nci_adapter_create_known_tag(self, ntf);
         }
         if (!tag) {
-            nfc_adapter_add_other_tag(NFC_ADAPTER(self), self->target);
+            nfc_adapter_add_other_tag2(NFC_ADAPTER(self), self->target, NULL);
```

Comment:
This isn't any better than it was before. The point of the deprecation warning was to make sure that the actual poll parameters are passed in (in cases when they make sense), not NULL.

Reply:
Thanks a lot, I have fixed that